### PR TITLE
Component tests

### PR DESCRIPTION
**Pull request by nikhilaravi**
added/updated tests for:
- package tile
- filter tile
- star rating
- tags

### CODE REVIEW

File: lib/filter-tile-yesno/index.js [hunk 1]
```diff
@@ -33,7 +34,8 @@ export default class FilterTile extends Component {
               showAddMessage={showAddMessage}
             />
           </div>
-        </div>}
+        </div>
+      }
```
Comment (nelsonic) [inline lib/filter-tile-yesno/index.js]: more readable. :+1:

File: lib/scroll-view/test/index.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// import React from 'react';
+// import { mount } from 'enzyme';
+// import jsdom from 'mocha-jsdom';
+// import ScrollView from '../';
```
Comment (nelsonic) [inline lib/scroll-view/test/index.test.js]: guessing we are going to come *back* to this later...?

**Comment by nelsonic** [comment]
looks good. :+1:

File: lib/scroll-view/test/index.test.js (hunk 2 above)
Comment (nikhilaravi) [inline lib/scroll-view/test/index.test.js]: Yes. I need to do some research about how to set the node heights etc.

**Pull request merged**